packages/material-ui/src/MenuList/MenuList.d.ts

```diff
@@ -24,6 +24,12 @@ export interface MenuListProps extends StandardProps<ListProps, MenuListClassKey
    * If `true`, the menu items will not wrap focus.
    */
   disableListWrap?: boolean;
+  /**
+   * If `true`, the MenuList is within a nested
+   * menu of a parent Menu, causing it to stop
+   * event propogation through the component
+   */
+  isSubMenu?: boolean;
```

Comment:
Unfortunately, there's no way to avoid passing this to MenuList via the `MenuListProps` for subMenus, since this is the only thing we're passing and it's needed to `stopPropagation()` of keyboard events in subMenus.